projects/listening-comp/services/audio-module/app.py

```
@@ -123,19 +174,25 @@ def process_questions(video_id):
-        # Process each question
+        # Process each question using OPEA TTS service
         for i, question in enumerate(questions):
             # Generate audio for the question
             question_text = question.get('question', '')
             if question_text:
                 try:
-                    # Call the TTS endpoint
-                    tts_response = text_to_speech()
-                    tts_data = tts_response.get_json()
+                    # Create TTS request
+                    tts_request = {
```

Ruff (0.8.2)

Local variable `tts_request` is assigned to but never used

Remove assignment to unused variable `tts_request`

(F841)